File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.py
class Solution:
    def getMinDistance(self, nums: List[int], target: int, start: int) -> int:
        ans = inf
        for i, num in enumerate(nums):
            if(num == target):
                if(abs(i - start) < ans):
                    ans = abs(i - start)
                else:
                    return ans
        
        return ans
        

File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.go
func getMinDistance(nums []int, target int, start int) int {
    res := 100_000
    for i, num := range nums {
        if num == target {
            if abs(i - start) < res {
                res = abs(i - start)
            }
        }
    }
    return res
}

func abs(diff int) int {
    if diff < 0 {
        return -1 * diff
    }
    return diff
}